.ruby-version

```diff
@@ -0,0 +1 @@
+2.2.3
```

Comment:
Okay, this might be seem small, but this is such good practice! I once had Heroku __blow up__ on me because I was using a built-in method that was in a newer version of Ruby. :+1: